src/main/java/io/github/pmckeown/dependencytrack/finding/FindingsMojo.java

```diff
@@ -75,33 +83,46 @@ public class FindingsMojo extends AbstractDependencyTrackMojo {
     private FindingsPrinter findingsPrinter;
     private FindingsAnalyser findingsAnalyser;
     private FindingsReportGenerator findingsReportGenerator;
+    private PolicyAction policyAction;
+    private PolicyViolationsPrinter policyViolationsPrinter;
+    private PolicyAnalyser policyAnalyser;
 
     @Inject
     public FindingsMojo(ProjectAction projectAction, FindingsAction findingsAction, FindingsPrinter findingsPrinter,
                         FindingsAnalyser findingsAnalyser, FindingsReportGenerator findingsReportGenerator,
-                        CommonConfig commonConfig, Logger logger) {
+                        CommonConfig commonConfig, Logger logger, PolicyAction policyAction,
+                        PolicyViolationsPrinter policyViolationsPrinter, PolicyAnalyser policyAnalyser) {
         super(commonConfig, logger);
         this.projectAction = projectAction;
         this.findingsAction = findingsAction;
         this.findingsPrinter = findingsPrinter;
         this.findingsAnalyser = findingsAnalyser;
         this.findingsReportGenerator = findingsReportGenerator;
+        this.policyAction = policyAction;
+        this.policyViolationsPrinter = policyViolationsPrinter;
+        this.policyAnalyser = policyAnalyser;
     }
 
     @Override
     protected void performAction() throws MojoExecutionException, MojoFailureException {
         List<Finding> findings;
+        List<PolicyViolation> policyViolations;
         try {
             Project project = projectAction.getProject(commonConfig.getProjectName(), commonConfig.getProjectVersion());
             findings = findingsAction.getFindings(project);
+            policyViolations = policyAction.getPolicyViolations(project);
             findingsPrinter.printFindings(project, findings);
-
+            policyViolationsPrinter.printPolicyViolations(project, policyViolations);
             boolean policyBreached = findingsAnalyser.doNumberOfFindingsBreachPolicy(findings, findingThresholds);
-            findingsReportGenerator.generate(getOutputDirectory(), findings, findingThresholds, policyBreached);
+            boolean policyViolationBreached = policyAnalyser.isAnyPolicyViolationBreached(policyViolations, policyConfig);
+            findingsReportGenerator.generate(getOutputDirectory(), findings, findingThresholds, policyBreached, policyViolations);
 
             if (policyBreached) {
                 throw new MojoFailureException("Number of findings exceeded defined thresholds");
             }
+            if(policyViolationBreached) {
```

Comment:
```suggestion
            if (policyViolationBreached) {
```

Sorry inconsistent white spacing triggers me ;)